integration_tests/src/test/kotlin/utils/CommandHelper.kt

```diff
@@ -4,10 +4,24 @@ import java.io.File
 import java.util.concurrent.TimeUnit
 
 fun String.runCommand(workingDir: File): ProcessResult = ProcessBuilder(*split("\\s".toRegex()).toTypedArray())
-    .directory(workingDir)
-    .redirectOutput(ProcessBuilder.Redirect.PIPE)
-    .redirectError(ProcessBuilder.Redirect.PIPE)
-    .start().run {
-        waitFor(60, TimeUnit.MINUTES)
-        ProcessResult(exitValue(), inputStream.bufferedReader().readText() + this.errorStream.bufferedReader().readText())
-    }
+        .directory(workingDir).run {
+            if (System.getProperty("os.name").startsWith("Windows"))
```

Comment:
I think we can use `ftl.config.FtlConstants.isWindows`?

Reply:
agreed will change

Reply:
Interestingly no, for now  this is inside of the integration_task subproject. It does not have access `ftl.config.FtlConstants.isWindows`